scripts/measures/2018/merge-measures-data.js

```diff
@@ -0,0 +1,21 @@
+const fs = require('fs');
+const path = require('path');
+
+const outputPath = process.argv[4];
+
+function mergeMeasures() {
+  const allJson = [];
+  // first command-line argument starts at position 2, last argument
+  // is the output path. everything in between is an input measures file
+  for (let i = 2; i < process.argv.length - 1; i++) {
+    const measureStagingPath = path.join(__dirname, process.argv[i]);
+    const measureJson = JSON.parse(fs.readFileSync(measureStagingPath, 'utf8'));
+    allJson.push(measureJson);
```

Comment:
I thought it wasn't necessary because two input files wouldn't have colliding measure ids (since they're provided by cms as separate quality, PI, IA measures files) buuuuut yeah it would be safer to check in case we screw up the command line arguments or something. interestingly json schema doesn't support this but AJV our validator library does (though only in a newer version than we use). i'll plan to add this in a separate PR!

Reply:
https://github.com/epoberezkin/ajv-keywords#uniqueitemproperties